tests/slist_ut/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(theseTestsName slist_ut)
+
+set(${theseTestsName}_test_files
+    ${theseTestsName}.c
+)
+
+set(${theseTestsName}_c_files
+    ../../src/slist.c
+)
+
+set(${theseTestsName}_h_files
+    ../../inc/c_util/slist.h
+)
+
+build_test_artifacts(${theseTestsName} "tests/c_util" ADDITIONAL_LIBS c_pal c_pal_reals)
```

Comment:
>) [](http://example.com/codeflow?start=87&length=1)

Let's have a newline at the end of the file.

Reply:
done